http/css/style.css

```diff
@@ -50,7 +50,23 @@ p { max-width: 800px; }
 
 .extension-child { padding-left: 1em; }
 .extension:hover { background-color: #babdb6; }
-.task { border-radius: 4px; padding-right: 4px; text-align: right; }
+.task {
+    position: relative;
+    border-radius: 4px;
+    padding-right: 4px;
+    height: 19px;
+    width: 72px;
+    padding: 0;
+}
+
+.task span {
+    position: absolute;
+    bottom: 0;
+    right: 0;
+    font: 9px sans-serif;
+    text-decoration: none;
+    color: black;
+}
```

Comment:
I try to use `em` instead of `px`, it is more browser-friendly and mobile-friendly, and generally considered a good practice. Although percent is a good practice too, but definitely not pixels which are too fixed.

http://kyleschaeffer.com/development/css-font-size-em-vs-px-vs-pt-vs/